pachyderm/templates/etcd/statefulset.yaml

```diff
@@ -80,7 +80,7 @@ spec:
       name: etcd-storage
       namespace: {{ .Release.Namespace }}
     spec:
-      {{- if not (eq .Values.pachd.storage.backend "LOCAL") }}
+      {{- if or .Values.etcd.storageClass  (eq .Values.deployTarget "GOOGLE") (eq .Values.deployTarget "MICROSOFT") (eq .Values.deployTarget "AMAZON") }}
```

Comment:
Would it be wrong to say `{{- if or .Values.etcd.storageClass (not eq .Values.deployTarget "LOCAL") }}`?

Reply:
So, I wanted to make it explicit incase we add more cloud providers (and to support the "Custom" deploy target)